### PR TITLE
Add rich text attribute input

### PR DESCRIPTION
**Pull request by piotrgrundas**
I want to merge this change because I would like to add simple text input attribute to the dashboard.

[API PR](https://github.com/mirumee/saleor/pull/7059/)

**PR intended to be tested with API branch:** 

### Screenshots
image
image

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [x] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Data-test are added for new elements.
1. [x] Type definitions are up to date.
1. [x] Changes are mentioned in the changelog.
1. [x] The changes are tested in different browsers (Chrome, Firefox, Safari).
1. [x] The changes are tested in light and dark mode.

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

File: src/components/Attributes/AttributeRow.tsx [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+          <TextField
+            fullWidth
+            disabled={disabled}
+            error={!!error}
+            label={intl.formatMessage(messages.valueLabel)}
+            value={attribute.value[0]}
+            helperText={getErrorMessage(error, intl)}
+            onChange={event => onChange(attribute.id, event.target.value)}
+          />
+        </BasicAttributeRow>
```
Comment (piotrgrundas) [inline src/components/Attributes/AttributeRow.tsx:271]: I've move that huge ternary into separate component and added this ☝️

File: src/components/Attributes/AttributeRow.tsx [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+import { ProductErrorWithAttributesFragment } from "@saleor/fragments/types/ProductErrorWithAttributesFragment";
+import { FormsetChange } from "@saleor/hooks/useFormset";
+import { ReorderEvent } from "@saleor/types";
+import { AttributeInputTypeEnum } from "@saleor/types/globalTypes";
+import { getProductErrorMessage } from "@saleor/utils/errors";
+import getPageErrorMessage from "@saleor/utils/errors/page";
+import React from "react";
+import { defineMessages, IntlShape, useIntl } from "react-intl";
+
+function getSingleChoices(
```
Comment (mmarkusik) [inline src/components/Attributes/AttributeRow.tsx:29]: All of these functions should be moved to utils

**Review by mmarkusik — CHANGES_REQUESTED** [review]

File: src/components/Attributes/AttributeRow.tsx (hunk 2 above)
Comment (piotrgrundas) [reply src/components/Attributes/AttributeRow.tsx:29]: I thought u'll yell if I mess too much with ur code 👀

**Review by jwm0 — CHANGES_REQUESTED** [review]

File: schema.graphql [hunk 3]
```diff
@@ -479,6 +479,7 @@ input AttributeFilterInput {
   filterableInStorefront: Boolean
   filterableInDashboard: Boolean
   availableInGrid: Boolean
+  metadata: [MetadataInput]
```
[the remark this answers is not in the document]
Comment (piotrgrundas) [reply schema.graphql]: Those metadata in inputs was just added, so from what I've checked & tested we are safe.

**Review by mmarkusik — APPROVED** [review]

**Review by jwm0 — APPROVED** [review]

**Review by jwm0 — APPROVED** [review]

**Pull request merged**